CMakeLists.txt

```diff
@@ -241,7 +240,7 @@ if(SDL2RTF_INSTALL)
 endif()
 
 if(SDL2RTF_SAMPLES)
-    add_executable(showrtf showrtf.c)
+    add_executable(showrtf samples/showrtf.c)
```

Comment:
`samples` -> `examples` maybe? (like other satellite libs.)

Reply:
I changed this.

Reply:
Off-topic: in SDL_mixer, playmus and playwave sources can be similarly moved under an `examples` directory, and `SDL_mixer.sym` can be moved under `src` directory, for better consistency.